sdk/translation/Azure.AI.Translation.Document/tests/samples/SampleSnippets.cs

```diff
@@ -71,21 +77,33 @@ public void BadRequestSnippet()
         }
 
         [Test]
-        [Ignore("Samples not working yet")]
+        [SyncOnly]
```

Comment:
This shouldn't be added. The sample is just showing inputs and there is no actual call to `client.StartTranslation` therefore, we don't need to actually call `Uri sourceSasUri = CreateSourceContainer(oneTestDocuments);`